db/migrate/20170403155048_create_jobs.rb

```diff
@@ -0,0 +1,9 @@
+class CreateJobs < ActiveRecord::Migration[5.0]
+  def change
+    create_table :jobs do |t|
+      t.datetime :last_checked_at
```

Comment:
I think a job will only run once so we probably just need a completed_at timestamp here